firstboot/firstboot.go

```diff
@@ -69,16 +69,19 @@ network:
 // be run as part of the config-changed hook and read the snap's
 // config to determine the netplan config to write.
 func InitialNetworkConfig() error {
+	// If the config is already present, don't overwrite it.  See
+	// https://bugs.launchpad.net/snappy/+bug/1623119.
+	if _, err := os.Stat(netplanConfigFile); err == nil {
```

Comment:
You may consider using osutil.FileExists() for this, but I guess your version is more go-ish ;)